hs_core/hydroshare/resource.py

```diff
@@ -525,7 +533,8 @@ def create_new_version_resource(ori_res, new_res, user):
         set_to_private = True
 
     # add files directly via irods backend file operation
-    utils.copy_resource_files_and_AVUs(ori_res.short_id, new_res.short_id, set_to_private)
+    utils.copy_resource_files_and_AVUs(ori_res.short_id, new_res.short_id, set_to_private,
```

Comment:
@alvacouch Yes, the record is in both Django database and iRODS AVU. You can query UserQuota from a User instance in Django database. iRODS AVU is for @mjstealey 's script to use for quota calculation. Let me know if this clears up your doubts or not.